.php_cs.dist

```diff
@@ -1,7 +1,7 @@
 <?php
 
 $finder = new PhpCsFixer\Finder();
-$config = new PhpCsFixer\Config('json-schema', 'json-schema style guide');
+$config = new PhpCsFixer\Config('json-schema');
```

Comment:
The name is still here, @erayd.

Reply:
My mistake, sorry! I looked at the diff and thought I saw the whole line removed.

Reply:
All good, @erayd, thank you!